packages/storage-adapter-local-forage/src/index.js

```diff
@@ -57,7 +57,12 @@ export default class StorageAdapterLocalForage {
         const key_ = `${namespaces.get(this)}/${key}`;
         loggers.get(this).info(`local-forage-store-adapter: reading \`${key_}\``);
         return localforage.getItem(key_)
-          .then((value) => value || Promise.reject(new NotFoundError(`No value found for ${key_}`)));
+          .then((value) => {
+            if (value !== null && value !== undefined) {
```

Comment:
localforage will return a null if no key is found, so this catches that case and throws